### PR TITLE
Grid list steps

### PR DESCRIPTION
**Pull request by nikola-bleech**
Update code to new standard: Refactor the grid styles (use grid mixin)

### CODE REVIEW

File: Components/GridListSteps/_style.scss [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
-  padding: $component-spacing-desktop 0;
+  padding: $component-spacing-mobile 0;
+
+  @media (min-width: $breakpoint-tablet) {
+    padding: $component-spacing-tablet 0;
+  }
+
+  @media (min-width: $breakpoint-desktop) {
+    padding: $component-spacing-desktop 0;
+  }
```
Comment (steffenbew) [inline Components/GridListSteps/_style.scss]: Could you use the class `.centerMaxWidthContainer` on the `.container` element instead? (like used in the BlockWysiwygSidebar component)

File: Components/GridListSteps/_style.scss [hunk 2]
```diff
@@ -15,29 +26,40 @@
...
+    @include reset-list;
 
-  .step {
-    max-width: 400px;
-    padding: 20px 10px 0;
+    align-items: center;
+    counter-reset: section;
+    flex-direction: column;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (steffenbew) [inline Components/GridListSteps/_style.scss]: I assume this is about centering the grid items. If so, could we make use of the mixin `$alignment` option?

**Comment by steffenbew** [comment]
In general: the changes add more CSS than previously, but the new mixins _should_ reduce complexity (besides harmonizing the approach). Could you please check if "it gets any simpler"?

File: Components/GridListSteps/_style.scss (hunk 1 above)
Comment (nikola-bleech) [reply Components/GridListSteps/_style.scss]: Will use the class as suggested.
Please note that `.centerMaxWidthContainer` class does not contain`$component-spacing-mobile` variable

Comment (nikola-bleech) [reply Components/GridListSteps/_style.scss]: Class `.centerMaxWidthContainer` is dealing with the horizontal spacing. 
Styles marked here are defining the vertical spacing of the components.

I will add `.centerMaxWidthContainer` class in the markup.

File: Components/GridListSteps/_style.scss (hunk 2 above)
Comment (nikola-bleech) [reply Components/GridListSteps/_style.scss]: `$alignment` option of the `grid` mixin deals only with `justify-content` property. 
Marked code, supplements the code generated by the mixin.

File: Components/GridListSteps/_style.scss (hunk 1 above)
Comment (steffenbew) [reply Components/GridListSteps/_style.scss]: ah sure, understand.

**Comment by nikola-bleech** [comment]
Revert the grid mixin to original styling.

**Pull request merged**